packages/client/src/components/FeaturedMovie.js

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import {
+  Box,
+  CardMedia,
+  Card,
+  CardContent,
+  IconButton,
+  Typography,
+} from '@mui/material';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+
+export default function FeaturedMovie() {
+  return (
+    <Card>
+      <Box sx={{ position: 'relative', display: 'flex' }}>
+        <CardMedia
+          component="img"
+          sx={{
+            width: 1672,
+            height: 790,
+            borderRadius: 395,
```

Comment:
How come such a high number for border radius?

Reply:
actually, these dimensions I took from the Figma design template for the FP. For me, they are high but I thought this is the way :D